src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
The unwind info format is different for each platform. The Windows ABI documents linked at the start of https://github.com/dotnet/coreclr/blob/master/Documentation/botr/clr-abi.md have the specs of unwind info format for x64 and ARM.

There is no standalone unwind info defined for Windows x86. Windows x86 ABI was defined before the concept of standalone unwind info came to Windows and nobody bothered to define it retroactively.

On Windows x86, the unwind info is encoded as part of GCInfo. The unwinder for it is at https://github.com/dotnet/coreclr/blob/master/src/vm/eetwain.cpp#L3897. For Linux 86, you have basically two options:

1. Keep using the existing scheme where the unwind info is encoded as part of GCInfo, maybe with some minimal modifications. These modifications have to be written down so that everybody is on the same page.
    or 
2. Define your own new x86 unwind info format that is similar to the unwind info formats used on non-x86 platforms. The spec for this format has to be written down so that everybody is on the same page.

I would recommend going with option 1, at least for now.